src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -287,6 +289,17 @@ public final class SunEntries {
          * Set the digest provider based on whether native crypto is
          * enabled or not.
          */
+        /* Disabling OpenSSL usage on AIX due to perfomance regression observed. */
```

Comment:
The comment should have the same tone as the one above it, suggest (with typo corrected):
```
        /* Don't use native MD5 on AIX due to an observed performance regression. */
```

Reply:
Changed.